plugins/formal-verification/testData/diagnostics/bad_contracts/binary_search.kt

```diff
@@ -0,0 +1,54 @@
+// USE_STDLIB
+
+import org.jetbrains.kotlin.formver.plugin.AlwaysVerify
+
+@AlwaysVerify
+fun <!VIPER_TEXT!>mid_increased_by_one<!>(arr: List<Int>, target: Int): Boolean {
+    val size = arr.size
+    val mid = arr.size / 2 + 1 // possible IndexOutOfBound
```

Comment:
I think the location/contents of these comments are a bit confusing.  The `IndexOutOfBounds` comes later and Viper already tells us about it.  I think it's good to point out which line has changed; perhaps something like "if arr.size == 1, mid is out of bounds" would be clearer?